apps/projects/app/components/Content/IssueDetail/EventsCard.js

```diff
@@ -21,7 +21,13 @@ const IssueEvent = ({ user, ...props }) => {
     <IssueEventMain>
       <div css="display: flex">
         <IssueEventAvatar>
-          <img src={user.avatarUrl} alt="user avatar" css="width: 40px; border-radius: 50%" />
+          <Link
```

Comment:
Why not use the new `Avatar` component here?

Reply:
Fixed.